tests/integration/helpers.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper functions for integration tests."""
+
+import typing
+
+import requests
+from juju.action import Action
+from juju.application import Application
+
+
+def create_moderators_room(
+    synapse_ip: str,
+    access_token: str,
+):
+    """Create "moderators" room in Synapse
+
+    Args:
+        synapse_ip: Synapse IP
+        access_token: Access token for Synapse to create the room
+
+    """
+    authorization_token = f"Bearer {access_token}"
+    headers = {"Authorization": authorization_token}
+    room_body = {
+        "creation_content": {"m.federate": False},
+        "name": "moderators",
+        "preset": "public_chat",
+        "room_alias_name": "moderators",
+        "room_version": "1",
+        "topic": "moderators",
+    }
+    res = requests.post(
+        f"http://{synapse_ip}:8080/_matrix/client/v3/createRoom",
+        json=room_body,
+        headers=headers,
+        timeout=5,
+    )
+    res.raise_for_status()
+
+
+def get_access_token(synapse_ip, user_username, user_password):
```

Comment:
```suggestion
def get_access_token(synapse_ip, user_username, user_password) -> str:
```

Reply:
Thanks for the advice. Fixed in another commit